webpack.config.js

```diff
@@ -78,7 +79,17 @@ module.exports = (env, argv) => {
       libraryTarget: 'umd',
       globalObject: 'this'
     },
-    plugins: []
+    plugins: [
+      // Can't get this regex exclusion to work...
+      // new webpack.IgnorePlugin(/\.\/wordlists\/(?!.*english.*)\.json/)
```

Comment:
I couldn't get this regex to work so just listed out all non-english files. LMK if you know how to fix this